apps/google-analytics-4/frontend/src/components/main-app/constants/noteMessages.ts

```diff
@@ -1,13 +1,13 @@
-export const DEFAULT_ERR_MSG = 'Oops! Cannot display the analytics data at this time.';
-export const EMPTY_DATA_MSG = 'There are no page views to show for this range.';
-export const DEFAULT_CONTENT_MSG = 'Oops! Something went wrong with the slug field configuration.';
+export const DEFAULT_ERR_MSG = 'An unknown error has occurred. Please try again or contact support';
```

Comment:
is there an optimal link to provide the user tp contact support? Thoughts on adding a hyperlink to https://www.contentful.com/support/?utm_source=webapp&utm_medium=help-menu&utm_campaign=in-app-help ?